packages/web-app-preview/src/App.vue

```diff
@@ -87,6 +88,7 @@ import MediaAudio from './components/Sources/MediaAudio.vue'
 import MediaImage from './components/Sources/MediaImage.vue'
 import MediaVideo from './components/Sources/MediaVideo.vue'
 import { CachedFile } from './helpers/types'
+import AppBanner from '../../web-pkg/src/components/AppBanner.vue'
```

Comment:
Packages should not import from other packages via relative paths.

```suggestion
import AppBanner from 'web-pkg/src/components/AppBanner.vue'
```